webui/react/src/App.tsx

```diff
@@ -1,36 +1,47 @@
 import { Button, notification } from 'antd';
 import React, { useEffect, useLayoutEffect, useState } from 'react';
 import { HelmetProvider } from 'react-helmet-async';
+import { GlobalHotKeys } from 'react-hotkeys';
 
 import { setupAnalytics } from 'Analytics';
 import Link from 'components/Link';
 import Navigation from 'components/Navigation';
 import Router from 'components/Router';
-import StoreProvider, { useStore } from 'contexts/Store';
+import StoreProvider, { StoreAction, useStore, useStoreDispatch } from 'contexts/Store';
 import { useFetchInfo } from 'hooks/useFetch';
 import useKeyTracker from 'hooks/useKeyTracker';
 import usePolling from 'hooks/usePolling';
 import useResize from 'hooks/useResize';
 import useRouteTracker from 'hooks/useRouteTracker';
 import useTheme from 'hooks/useTheme';
+import Omnibar from 'omnibar/Component';
 import appRoutes from 'routes';
 import { correctViewportHeight, refreshPage } from 'utils/browser';
 
 import css from './App.module.scss';
 import { paths } from './routes/utils';
 
+const globalKeymap = { SHOW_OMNIBAR: [ 'ctrl+space' ] };
+
 const AppView: React.FC = () => {
   const resize = useResize();
-  const { info } = useStore();
   const [ canceler ] = useState(new AbortController());
+  const { info, omnibar } = useStore();
+  const storeDispatch = useStoreDispatch();
 
   const fetchInfo = useFetchInfo(canceler);
 
   useKeyTracker();
+  const globalKeyHandler = {
+    SHOW_OMNIBAR:
+    (): void => storeDispatch({ type: StoreAction.ShowOmnibar }),
+  };
+
   useRouteTracker();
   useTheme();
 
   // Poll every 10 minutes
+
```

Comment:
lint will catch this I'm sure

Reply:
I don't think we have such a lint rule. I removed the added line